client/src/components/InputTime.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import './InputTime.less';
+import { TimePicker } from 'antd';
+import moment from 'moment';
+import { useState } from 'react';
+import CallPaneCaller from '../components/CallPaneCaller';
+import 'antd/dist/antd.css';
+import FadeIn from 'react-fade-in';
+import { Button } from 'antd';
+
+function InputTime() {
+
+  const [makeCall, setMakeCall] = useState(false)
+  const [callLength, setCallLength] = useState('')
+
+  const handleOnSubmit = () => {
+    setMakeCall(true)
+  }
+
+  const handleOnChange = (time, timeString) => {
+    setCallLength(timeString)
+  }
+
+  if (!makeCall) {
```

Comment:
Hidden in this if, there's a decision on what is to make a call and how it works. This component is a container and not a presentational (it should be the second one).

Being a presentational, it should take the event and pass it up to the closest container so it runs the business logic.

Reply:
Correct and agreed - will refactor